CovertInfiltration/Src/CovertInfiltration/Classes/X2StrategyElement_InfiltrationActions.uc

```diff
@@ -0,0 +1,117 @@
+// 
+
+class X2StrategyElement_InfiltrationActions extends X2StrategyElement;
+
+static function array<X2DataTemplate> CreateTemplates()
+{
+	local array<X2DataTemplate> CovertActions;
+	
+	//CovertActions.AddItem(CreatePrepareGOpTemplate());
```

Comment:
Should this be enabled later? If so, add a `// FIXME: Re-enable later` or something. Otherwise remove.